db/people.yml

```diff
@@ -3106,10 +3106,19 @@ vgharakhanyan:
   position: Graduate Student
 vlin:
   active: true
-  aka: []
+  aka: Viv
```

Comment:
these aka's are used for linking you to your publications etc., so if there is a paper that is published where you appear as "Lin, V." we wouldn't be able to correctly dereference it to your profile here where you appear as either vlin or "Vivian Lin".  Just leave this as an empty list until this becomes an issue! ;)

Reply:
fixed!